src/client/App.js

```diff
@@ -46,7 +46,7 @@ function App() {
             <Profile />
           </AuthenticatedRoute>
           <Route exact path="/overview">
-            <Overview />
+            <Overview roomList={[]} />
```

Comment:
You shouldn't pass any parameters in this file. We only mention components for routing here.

Reply:
i see. ive removed it.